Run failed: Traceback (most recent call last): File "/var/sandbox/sandbox-python/tmp/d8cf63d0_2472_4d95_bfc4_17cea97e345d.py", line 48, in <module> File "<string>", line 23, in <module> File "<string>", line 14, in main File "/usr/local/lib/python3.10/site-packages/jinja2/environment.py", line 1291, in render self.environment.handle_exception() File "/usr/local/lib/python3.10/site-packages/jinja2/environment.py", line 925, in handle_exception raise rewrite_traceback_stack(source=source) File "<template>", line 3, in top-level template code TypeError: 'builtin_function_or_method' object is not iterable error: exit status 255